layersvt/device_simulation.cpp

```diff
@@ -236,6 +424,24 @@ std::mutex global_lock;  // Enforce thread-safety for this layer's containers.
 uint32_t loader_layer_iface_version = CURRENT_LOADER_LAYER_INTERFACE_VERSION;
 
 typedef std::vector<VkQueueFamilyProperties> ArrayOfVkQueueFamilyProperties;
+typedef std::unordered_map<uint32_t /*VkFormat*/, VkFormatProperties> ArrayOfVkFormatProperties;
```

Comment:
Why not use VkFormat as the key?

Reply:
My Ubuntu 16.04's gcc 5.4 doesn't like it.
Apparently it's fixed in [gcc 6.1](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=60970)